src/Native/Unix/System.Native/pal_networkstatistics.h

```diff
@@ -167,6 +167,8 @@ DLLEXPORT int32_t SystemNative_GetEstimatedUdpListenerCount(void);
 
 DLLEXPORT int32_t SystemNative_GetActiveUdpListeners(IPEndPointInfo* infos, int32_t* infoCount);
 
+DLLEXPORT uint32_t SystemNative_GetNativeIPInterfaceIndex(char* interfaceName);
```

Comment:
Why pal_networkstatistics.c/h?  Seems like it belongs better in pal_networking.c/h.

Reply:
Makes sense